src/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-weight: 300px;
+}
+
+.search__barBig {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 2.5%;
+  background-size: 19px 19px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+#searchB {
+  text-indent: 62px;
+}
+
+.search__barBig:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search__barBig:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+/* Small Sewarch Bar */
+
+.search__barSmall {
+  display: flex;
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 1.5%;
+  background-size: 11px 11px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+#searchS {
+  text-indent: 33px;
+}
+
+.search__barSmall:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__barSmall:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)